cmd/spire-server/cli/token/generate.go

```diff
@@ -64,6 +64,8 @@ func (g GenerateCLI) Run(args []string) int {
 			fmt.Printf("Error assigning SPIFFE ID: %s\n", err.Error())
 			return 1
 		}
+	} else {
```

Comment:
it can be simplified:
```
if config.SpiffeID == "" {
   fmt.Printf("Warning: Missing SPIFFE ID.\n")
   return 0
}
 err = g.createVanityRecord(ctx, c, token, config.SpiffeID)
 if err != nil {
	fmt.Printf("Error assigning SPIFFE ID: %s\n", err.Error())
	return 1
 }
```